### PR TITLE
zx81_cass.xml: Added 10 working items + 1 not working

### PR DESCRIPTION
**Pull request by ArcadeShadow**
New working software list additions
--------------------------------------------
3D Formule 1 [Team Zeddy, Planeta Sinclair]
AERI - Análise de Estruturas Recticuladas Isostácticas [Planeta Sinclair]
Alien Invaders [Team Zeddy, Planeta Sinclair]
Air Traffic Control [Team Zeddy, Planeta Sinclair]
Colunas - Dimensionamento de Colunas Montantes de Edifícios [Planeta Sinclair]
Dis-ACSem [Planeta Sinclair]
Em Busca dos Tesouros [Spectrum Computing]
Geometria I [Planeta Sinclair]
Letter Hunt [Planeta Sinclair]
Tut-Tut [Spectrum Computing]

New NOT working software list addition
--------------------------------------------
1K Games Pack (Crystal) [Team Zeddy, Planeta Sinclair]

### CODE REVIEW

**Comment by ArcadeShadow** [comment]
I couldn't get the "1K Games Pack (Crystal)" to work but it could be something I did incorrectly.
I transcribed the loading instructions to the set's notes.
image

**Comment by ICEknigh7** [comment]
Tried setting the emulated ZX81 to 1K RAM just in case but can't get them to work either, guess it's just a MAME bug since Planeta Sinclair could run them fine with whatever emulator they're using.

File: hash/zx81_cass.xml [hunk 1]
```diff
@@ -958,7 +1060,7 @@ TZX files for ZX81 are just the .P dumps with an additional header on top, so th
 			</dataarea>
 		</part>
 	</software>
-
+
```
Comment (cuavas) [inline hash/zx81_cass.xml:1063]: Stray tab snuck in.

File: hash/zx81_cass.xml [hunk 2]
```diff
@@ -8693,4 +8878,20 @@ TZX files for ZX81 are just the .P dumps with an additional header on top, so th
...
+		<year>2019</year>
+		<publisher>zx81keyboardadventure.com</publisher>
+		<info name="developer" value="David A. Stephenson" />
+		<info name="usage" value="16K" />
+		<part name="cass1" interface="zx81_cass">
+			<dataarea name="cass" size="15746">
+				<rom name="tut-tut.p" size="15746" crc="3af8b76f" sha1="228b11560f2937d7dfed0edc4b120fada36335f8"/>
+			</dataarea>
+		</part>
+	</software>
```
Comment (cuavas) [inline hash/zx81_cass.xml:8895]: Just use the name of the site as they write it in the header and footer – “ZX81 Keyboard Adventure”.

**Review by Pernod70 — APPROVED** [review]
No further comments from me.

**Comment by ArcadeShadow** [comment]
@cuavas fixes done

**Pull request merged**